registers/TUTORIAL.md

```diff
@@ -19,3 +19,45 @@ You can run it with
 By default it uses the ["register" register (a list of all
 registers)](https://register.register.gov.uk/) but you can point it at
 others with the `--register` flag.
+
+Step 2
+------
+
+Next, we are going to put the data into a Trillian log. The first step
+is run the log server and create a log.
+
+Build Trillian:
+
+```make trillian```
+
+Run the log server (this will tie up a terminal window):
+
+```make tlserver```
+
+And create a log (if you need to redo this, you can run `make
+deletelog` first):
+
+```make createlog```
```

Comment:
Similarly, having the generated tree ID be secretly written to the logid file might confuse people who are following this for the first time.

I think it would make more sense to explicitly say this will print out the new tree ID and explain how it's used later.